server/routes/core.ts

```diff
@@ -8,6 +8,11 @@ import { log } from '../log';
 
 const router = Router();
 
+router.get('/csp-audit-report-endpoint', (req: Request, res: Response) => {
+	log.info(`CSP_AUDIT: ${req.body}`);
```

Comment:
sometimes cloudwatch is a bit irritating and splits multi line log statements across more than one log line.  It does have smarts to spot json, but if we hit this problem, we can always indent subsequent lines (which should help)